README.md

```diff
@@ -34,9 +34,30 @@ ownership of their social media presence.
 
 ## Development
 
+<a href="https://gitpod.io/github.com/TechSquidTV/Shoutify">
+  <img
+    src="https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod"
+    alt="Contribute with Gitpod"
+  />
+</a>
+
 To setup and install Shoutify for development, please read the [HACKING.md](.github/HACKING.md) document.
 
+<<<<<<< Updated upstream
 ### Contributing
+=======
```

Comment:
Please remove this line (`48`) to line `60` and also `46`. It's your stashed commit and it was deleted in the last PR.
Check the [README](https://github.com/TechSquidTV/Shoutify/blob/main/README.md) here

Reply:
Ok, thanks I was wondering which bit was wrong XD

Reply:
Fixed